### PR TITLE
Status icons for Services visible on list and resources pages

### PR DESCRIPTION
**Pull request by digitalfishpond**
Services have status icons showing whether it is ready or still pending (waiting for a clusterIP), also situations where the Service card has no labels or no IP to display for whatever reason, the card displays a "-" instead of returning an error.

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/dashboard/809)

### CODE REVIEW

**Comment by digitalfishpond** [comment]
PTAL

<details><summary><i>Previously, codecov-io wrote…</i></summary>
> ## [Current coverage][cc-pull] is **94.79%**
> > Merging [#809][cc-pull] into [master][cc-base-branch] will increase coverage by **0.12%**
> 
> 1. 2 files (not in diff) in `...tend/common/resource` were modified. [more](https://codecov.io/gh/kubernetes/dashboard/commit/1ed8ddcae5670a2321511959752c24843f85d14b/changes?src=pr#7372632F6170702F66726F6E74656E642F636F6D6D6F6E2F7265736F75726365) 
>   - Misses `+1` 
>   - Hits `-13`
> 1. 1 files (not in diff) in `.../app/frontend/common` were modified. [more](https://codecov.io/gh/kubernetes/dashboard/commit/1ed8ddcae5670a2321511959752c24843f85d14b/changes?src=pr#7372632F6170702F66726F6E74656E642F636F6D6D6F6E) 
>   - Hits `-1`
> 1. 1 files (not in diff) in `src/app/frontend` were modified. [more](https://codecov.io/gh/kubernetes/dashboard/commit/1ed8ddcae5670a2321511959752c24843f85d14b/changes?src=pr#7372632F6170702F66726F6E74656E64) 
>   - Misses `-6` 
>   - Hits `+6`
> 1. 2 files (not in diff) in `...tend/common/resource` were deleted. [more](https://codecov.io/gh/kubernetes/dashboard/commit/1ed8ddcae5670a2321511959752c24843f85d14b/changes?src=pr#7372632F6170702F66726F6E74656E642F636F6D6D6F6E2F7265736F75726365)
> 1. 1 files (not in diff) in `.../app/frontend/common` were deleted. [more](https://codecov.io/gh/kubernetes/dashboard/commit/1ed8ddcae5670a2321511959752c24843f85d14b/changes?src=pr#7372632F6170702F66726F6E74656E642F636F6D6D6F6E)
> 
> ```diff
> @@             master       #809   diff @@
> ==========================================
>   Files           205        202     -3   
>   Lines          1674       1630    -44   
>   Methods           0          0          
>   Messages          0          0          
>   Branches          0          0          
> ==========================================
> - Hits           1584       1545    -39   
> + Misses           90         85     -5   
>   Partials          0          0          
> ```
> 
> > Powered by [Codecov](https://codecov.io?src=pr). Last updated by [13713df...1ed8ddc][cc-compare]
> [cc-base-branch]: https://codecov.io/gh/kubernetes/dashboard/branch/master?src=pr
> [cc-compare]: https://codecov.io/gh/kubernetes/dashboard/compare/13713dfa116f2b7da3e463fe970121d702c036ad...1ed8ddcae5670a2321511959752c24843f85d14b
> [cc-pull]: https://codecov.io/gh/kubernetes/dashboard/pull/809?src=pr
</details>


---

Review status: 0 of 9 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/809#-:-KJa14xG0quQKX1S-JQJ:676965300)*

**Comment by bryk** [comment]
Can you add tests for `servicecardlist_component.js`? 

<details><summary><i>Previously, digitalfishpond (Denis Poisson) wrote…</i></summary>
> PTAL
</details>


---

Reviewed 1 of 3 files at r1, 8 of 8 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/809#-:-KJdfyR-DsAE8_FWhWQb:-56804714)*

**Comment by bryk** [comment]
Otherwise looks perfect

<details><summary><i>Previously, bryk (Piotr Bryk) wrote…</i></summary>
> Can you add tests for `servicecardlist_component.js`? 
</details>


---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/809#-:-KJdg6Z0wEzTLmHN5Rbk:-2073611408)*

File: src/app/frontend/servicelist/servicecardlist.html [hunk 1]
```diff
@@ -27,6 +27,15 @@
 
   <kd-resource-card ng-repeat="service in ::$ctrl.services"
       object-meta="service.objectMeta" type-meta="service.typeMeta">
+    <kd-resource-card-status layout="row">
+      <md-icon class="material-icons" ng-if="::$ctrl.isPending(service)">
+        timelapse
+        <md-tooltip md-direction="right">{{::$ctrl.i18n.MSG_POD_IS_PENDING_TOOLTIP}}</md-tooltip>
+      </md-icon>
+      <md-icon class="material-icons kd-success" ng-if="::$ctrl.isSuccess(service)">
+        beenhere
```
Comment (maciaszczykm) [inline src/app/frontend/servicelist/servicecardlist.html]: Please use `check_circle` instead.

**Pull request closed without merging**